backend/onedrive/onedrive.go

```diff
@@ -432,14 +480,18 @@ func (f *Fs) readMetaDataForPathRelativeToID(ctx context.Context, normalizedID s
 // readMetaDataForPath reads the metadata from the path (relative to the absolute root)
 func (f *Fs) readMetaDataForPath(ctx context.Context, path string) (info *api.Item, resp *http.Response, err error) {
 	firstSlashIndex := strings.IndexRune(path, '/')
-
-	if f.driveType != driveTypePersonal || firstSlashIndex == -1 {
+	if (f.driveType != driveTypePersonal && f.opt.Region != "cn") || firstSlashIndex == -1 {
 		var opts rest.Opts
 		if len(path) == 0 {
 			opts = rest.Opts{
 				Method: "GET",
 				Path:   "/root",
 			}
+		} else if f.opt.Region == "cn" {
```

Comment:
This new branch can be removed. We only need to use `:/` for the global version, or `/children/` for the Chinese version, in the next `else` branch.